app/views/layouts/wizard.html.slim

```diff
@@ -31,40 +31,49 @@ html[dir="#{@locale_direction}" lang="#{I18n.locale}" xml:lang="#{I18n.locale}"]
 
     = csrf_meta_tags
 
-  body.basic
-
+  body
+    
     = render 'layouts/header'
 
     #content
+      = render 'layouts/subheader'
       = render 'shared/flash_messages'
-      #app.inner
-        .row.text-center
-          .col-md-12
-            .panel.panel-default
-              ul.nav.nav-pills.nav-justified.alert-warning
-                li class=(params[:action] == "select_device" ? "active" : "disabled")
-                  a href="#"
-                    |Select new drive
-                    span.glyphicon.glyphicon-chevron-right
-                li class=(params[:action] == "select_fs" ? "active" : "disabled")
-                  a href="#"
-                    |Choose file system
-                    span.glyphicon.glyphicon-chevron-right
-                li class=(params[:action] == "manage_disk" ? "active" : "disabled")
-                  a href="#"
-                    | Options
-                    span.glyphicon.glyphicon-chevron-right
-                li class=(params[:action] == "confirmation" ? "active" : "disabled")
-                  a href="#"
-                    | Confirmation
-                    span.glyphicon.glyphicon-chevron-right
-                li class=(params[:action] == "done" ? "active" : "disabled")
-                  a href="#"
-                    | Done
-                    span.glyphicon.glyphicon-ok
-              hr/
-              = yield
+      #app.inner.container
+        - unless @no_tabs
+          /= render 'shared/tabs'
```

Comment:
In tabs layout, none of the tabs is getting activated. Ideally, discs tab should be activated. so temporary I have commented this line.